docs/docsite/rst/windows_winrm.rst

```diff
@@ -657,6 +666,34 @@ for each authentication option. See the section on authentication above for more
     encryption done over TLS. The WinRM payload is still encrypted with TLS
     when run over HTTPS, even if ``ansible_winrm_message_encryption=never``.
 
+HTTPS Certificate Validation
+````````````````````````````
+As part of the HTTPS protocol, the certificate is validated to ensure the host
+matches the subject and the client trusts the issuer of the server certificate.
+When using a self-signed certificate or setting
+``ansible_winrm_server_cert_validation: ignore`` these security mechanisms are
+bypassed. While self signed certificates will always need the ``ignore`` flag,
+certificates that have been issued from a certificate authority can still be
+validated.
+
+One of the more common ways of setting up a HTTPS listener in a domain
+environment is to use Active Directory Certificate Service (AD CS). AD CS is
+used by Active Directory to sign certificate requests that are sent to it and
+then issue a certificate from the csr. The issuer certificate of the AD CS
+instance can then be exported as a PEM encoded certificate. This file can then
+be copied locally to the Ansible controller.
+
+This file can either contain the single issuer certificate or multiple
+certificates. To then use this file as part of the valiation process, set
+``ansible_winrm_ca_trust_path`` to the path of the. If this value is not set
```

Comment:
Please either remove the word “which” or add parenthesis around that part of the sentence so it makes it easier to understand.